samples/get-started/.github/workflows/get-started.yml

```diff
@@ -15,14 +18,27 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
-        node-version: 16
+        node-version: 18
+     # This step is to sign-in to Azure to run tests from GitHub Action workflow, https://docs.github.com/en/actions/security-for-github-actions/security-hardening-your-deployments/configuring-openid-connect-in-azure
+    - name: OIDC Login to Azure Public Cloud with AzPowershell (enableAzPSSession true)
```

Comment:
https://learn.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-portal%2Clinux#use-the-azure-login-action-with-openid-connect this is more comprehensive guide to setup azure auth from github action, basically it support 2 method and one of them is OIDC but second method via [service connection](https://learn.microsoft.com/en-us/azure/developer/github/connect-from-azure-secret) is also available, its possible that azure customer are comfortable with the old one hence lets mention here